File: RAI/Clase15/Python/Vehiculo.py
class Vehiculo:

    def __init__(self, patente= None, marca = None, modelo = None):
        if patente != None:
            self.patente = patente
        else:
            self.patente = ""
        self.marca = marca if marca != None else ""
        self.modelo = modelo if modelo != None else ""
    
    def __str__(self):
        return "Vehiculo Patente: " + self.patente + " marca: " + self.marca + " modelo: " + (self.modelo if self.modelo != None else "")

File: RAI/Clase15/Java/Vehiculo.java
public class Vehiculo{
    private String patente;
    private String marca;
    private String modelo;

    public Vehiculo(String patente, String marca, String modelo){
        this.patente = patente;
        this.marca = marca;
        this.modelo = modelo;
    }

    public Vehiculo(String marca, String modelo){
        this.patente = "";
        this.marca = marca;
        this.modelo = modelo;
    }

    public Vehiculo(){
        this.patente = "";
        this.marca = "";
        this.modelo = "";
    }

    @Override
    public String toString(){
        return "Vehiculo patente: " + this.patente + " marca: " + this.marca + " modelo: " + this.modelo;
    }

    public String toString(int n){
        if(n == 1)
            return "Vehiculo patente: " + this.patente + " marca: " + this.marca + " modelo: " + this.modelo;
        else
            return "Vehiculo sin informacion: ";
    }
}